client/src/components/common/board/SearchBoard.jsx

```
@@ -47,17 +47,24 @@ export default function SearchBoard({ clickHandler }) {
+              return (
+                <SearchResultItem
+                  key={id}
+                  id={id}
+                  title={title}
+                  keyword={inputTitle}
+                  clickHandler={clickHandler}
+                ></SearchResultItem>
+              );
+            })}
+          </>
+        ) : (
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)